migrations/scripts/3_WitnetBytecodes.js

```diff
@@ -49,7 +49,7 @@ module.exports = async function (deployer, network, [, from]) {
         // deploy instance only if not found in current network:
         utils.traceHeader("Singleton inception of 'WitnetBytecodes':")
         const balance = await web3.eth.getBalance(from)
-        const gas = singletons.WitnetBytecodes.gas
+        const gas = singletons.WitnetBytecodes.gas || 5000000
```

Comment:
Can we get the 5000000 declared as a constant somewhere and import upon usage?

Reply:
Typo, it should be removed